lib/VM/JSLib/Intl.cpp

```diff
@@ -35,8 +37,15 @@ namespace {
 CallResult<std::u16string> stringFromJS(
     Runtime *runtime,
     PseudoHandle<> value) {
+  Handle<> valueHandle = runtime->makeHandle(std::move(value));
+  TagKind kind = valueHandle.get().getTag();
+  if(kind != StrTag && kind != ObjectTag ) {
```

Comment:
Please avoid directly comparing tags like this unless absolutely necessary.

Use the `isString()` and `isObject()` functions instead. (`valueHandle->isString()` will do the trick)

Reply:
Thanks!